packages/contracts/test/dollar/ERC20Ubiquity.t.sol

```diff
@@ -204,7 +204,7 @@ contract ERC20UbiquityTest is LocalTestHelper {
         address burner = address(0x2);
         vm.prank(admin);
         UbiquityDollarManager(dollar_manager_addr).grantRole(
-            keccak256("GOVERNANCE_TOKEN_BURNER_ROLE"), burner
+            keccak256("UBQ_BURNER_ROLE"), burner
```

Comment:
Refer to changed role names comments above.

Reply:
Fixed